server/src/main/java/org/elasticsearch/action/fieldcaps/FieldCapabilitiesResponse.java

```diff
@@ -41,6 +45,17 @@ public class FieldCapabilitiesResponse extends ActionResponse implements Chunked
     public static final ParseField FAILURES_FIELD = new ParseField("failures");
 
     private final String[] indices;
+    // Locally resolved index expressions for the current project.
+    // This is sent over the wire from linked projects to the coordinating project of a request to
+    // inform the coordinating project of the local resolution state (for each remote).
+    private final ResolvedIndexExpressions resolvedLocally;
```

Comment:
I wish there was a better way to name / explain this . It may be me, but when I read local here, I don't understand whether it's local to the linked projects or to the coordinating project.